.github/workflows/release-build-outputs.yml

```diff
@@ -0,0 +1,163 @@
+# HMS Networks Solution Center
+# Build Outputs Action for Maven-based Ewon ETK Project Releases
+# Version: 1.0
+#
+# This action is configured to automatically run when a release
+# tag is created in the following syntax: `v*`.
+
+name: Release (Build Outputs)
+
+on:
+  push:
+    tags:
+      - 'v*' # Push events to matching v*, i.e. v1.0
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  CHANGELOG_FILE_NAME: ./docs/CHANGELOG.md
+  RELEASE_INCLUDED: CHANGELOG_FILE_NAME README.md LICENSE src .classpath .project pom.xml build.xml
```

Comment:
Should the jvmrun file be included in the release .zip? Or is it documented that the jvmrun file is only available directly on the GitHub Release page.

Reply:
With the goal of syncing up with other connectors do most others refer users to the github page in documentation or include it in the release? I rather do what is most common in other connectors for consistency. If the most common is to have it documented, that would be in the short documentation PR i'll put out tomorrow

Reply:
Actually, the ability to include the `jvmrun` file directly in the GitHub Release is brand new (less than a month, see https://github.com/hms-networks/sc-java-maven-starter-project/commit/afd9a946daeba33f74a576835050b14ba2df73c3). At this point, everything is still referencing the jvmrun file that would be found in the release .zip

Reply:
I believe at this point only two other connector code bases are using this updated action with `jvmrun` and `.jar` directly in the GitHub Release page. I think for consistency and convenience, having in both locations might be good, but documentation need only reference or direct the user to one of those locations.